packages/web-twig/src/Resources/components/Toast/ToastBarLink.twig

```diff
@@ -0,0 +1,28 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _color = props.color | default('inverted') -%}
+{%- set _href = props.href -%}
+{%- set _isDisabled = props.isDisabled | default(false) -%}
+{%- set _isUnderlined = props.isUnderlined ?? true -%}
+{%- set _children = block('content') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'ToastBar__link' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _allowedAttributes = [ 'target', 'title' ] -%}
+{%- set _classNames = [ _rootClassName, _styleProps.className  ] -%}
+
+
+{% embed "@spirit/link.twig" with { props: props | merge({
+    color: _color,
+    href: _href,
+    isDisabled: _isDisabled,
+    isUnderlined: _isUnderlined,
+    UNSAFE_className: _classNames | join(' '),
+    UNSAFE_style: _styleProps.style,
```

Comment:
btw do we need to do this? And the same for href? Because you merge to props and don't modify it.

Reply:
You are right. But I think that we still need isUnderlined, because it change default value to true (Link is false by default). But other props are redundant.